scss/tech.scss

```diff
@@ -22,14 +22,19 @@ $tech-logo-size: 100px;
   position: relative;
   width: 100%;
 
+  &:hover::before {
+      top: 5px;
+      left: 5px;
+    }
```

Comment:
Deleite space before curly brace. I think content has extra indent

Reply:
Will be fixed on the next commit :)